FlowCrypt/Core/Core.swift

```diff
@@ -87,7 +87,7 @@ actor Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
         }
 
         let decrypted = try await call("decryptFile", jsonDict: [
-            "keys": try keys.map { try $0.toJsonEncodedDict() },
+            "keys": keys.map { $0.prvKeyInfoJsonDictForCore },
```

Comment:
Such `map` can be simplified to
```swift
keys.map(\.prvKeyInfoJsonDictForCore)
```

There are some other places in this PR where it can be used too.